Assets/Drag.cs

```diff
@@ -10,23 +10,52 @@ public class Drag : MonoBehaviour
     Vector3 startingPosition;
     GameObject clone;
     Boolean triggerFlag;
```

Comment:
rename trigger flag to something more descriptive, maybe `stillInContactWithOriginal` or something like that